RHEcosystemAppEng/dbaas-operator
# [DBAAS-939] Revert the api and change the code to support upgrade for 0.3.1

Signed-off-by: Abdul Hameed <ahameed@redhat.com>

## Description

https://issues.redhat.com/browse/DBAAS-939

## Verification Steps
Verify the name and types status fields should be same as 0.2.0, we have added conditions field in status, which also need verify is not making any issue while upgrading.

The filed name[
<img width="1142" alt="Screen Shot 2022-10-11 at 8 41 37 AM" src="https://user-images.githubusercontent.com/41960152/195093560-7607195f-12b6-4a7d-900d-0c88ba5f2b65.png">
](url)



## Type of change


## Checklist (Definition of Done)
